pages/science/reactors.rst

```diff
@@ -322,26 +322,30 @@ to the reactors previously mentioned:
   detail of how the wall affects the connected reactors.
 
 - :py:class:`Valve`: A valve is a flow devices with mass flow rate that is a function of
-  the pressure drop across it. The default behavior is linear:
+  the pressure drop across it. The mass flow rate is computed as:
 
   .. math::
 
-     \dot m = K_v (P_1 - P_2)
+     \dot m = c g(t) f(\Delta P)
```

Comment:
I would prefer to keep this explicitly as `P_1 - P_2`, since `\Delta` usually implies `2 - 1`. I see you have the definition later, but it's still a little counter intuitive to me.

Reply:
True. I'm following code here, which uses the opposite sign - this should probably be changed to be in line with conventions (I *think* this choice predates my updates). As an aside, the parameter names have been changed in the docstrings also, as they have a more generic meaning after the code change (i.e. coefficients are now defined as `m_coeff` in the `FlowDevice` class).